websocket-sink/README.md

```diff
@@ -0,0 +1,127 @@
+## Netty based Websocket Sink for Spring Cloud Data Stream
+SCDS version of [springxd-netty-websocket-sink](https://github.com/olmoser/spring-xd-netty-websocket-sink).
```

Comment:
Please avoid reference to individual github repositories for the code that gets committed into the main repo.